UnitTests/UnitTests.UWP/IsExternalInit.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    internal static class IsExternalInit
```

Comment:
Why is this here? According to [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.isexternalinit?view=net-5.0) we shouldn't use that name should we?🤔

Reply:
Is it worth adding a comment about this then? Or do we expect people to know intuitively why this is here/why it is not a bad idea?

Reply:
I'll add a comment. It would be more clear than the having something in the commit message. Thanks @chingucoding!

Reply:
I added a bit of an explanation and some useful links on the topic.